FESS-GUI/mainwindow.ui

```diff
@@ -280,7 +279,7 @@ Monitor</string>
               <number>3</number>
              </property>
              <item alignment="Qt::AlignLeft|Qt::AlignBottom">
-              <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
+              <widget class="QDoubleSpinBox" name="jerSpinBox">
```

Comment:
Let's try to spell out our words, instead of shortening them as we've been doing.